core/conversion/converters/impl/max.cpp

```diff
@@ -0,0 +1,47 @@
+#include "NvInfer.h"
+#include "core/conversion/converters/converters.h"
+#include "core/conversion/tensorcontainer/TensorContainer.h"
+#include "core/util/prelude.h"
+#include "torch/torch.h"
+
+#include <ATen/ATen.h>
+#include <vector>
+
+namespace torch_tensorrt {
+namespace core {
+namespace conversion {
+namespace converters {
+namespace impl {
+namespace {
+auto max_registrations TORCHTRT_UNUSED = RegisterNodeConversionPatterns().pattern(
+        {"aten::max.dim(Tensor self, int dim, bool keepdim=False) -> (Tensor values, Tensor indices)",
+         [](ConversionCtx* ctx, const torch::jit::Node* n, args& args) -> bool {
+             auto self = args[0].ITensorOrFreeze(ctx);
+             auto k = 1;
+             auto dim = args[1].unwrapToInt();
+             auto largest = true;
+             auto selfDim = util::toVec(self->getDimensions());
+             if (dim < 0) {
+                 dim = selfDim.size() + dim;
+             }
+             uint32_t shiftDim = 1 << dim;
+
+             auto TopKOperation = largest ? (nvinfer1::TopKOperation::kMAX) : (nvinfer1::TopKOperation::kMIN);
```

Comment:
you can remove auto largest = true and change this line to `auto TopKOperation = largest ? (nvinfer1::TopKOperation::kMAX)` since we are only using max always here